### DISCUSSION TITLE
Issues with thehive

### DISCUSSION THREAD

**Discussion by danmessner1** [Security-Onion-Solutions/securityonion#2253] [General] [answered]
I am FINALLY running 2.3.10

Setup is Dell PE R240 on ESXi 6.7

The ISO image never loaded for me. I saw a couple posts where it got to 79% and just hung. That is exactly what I saw. I finally loaded CENTOS7 with recommended partitions then downloaded the image via git. I suppose I could have done the same thing with the stock ISO but ultimately what worked for me is to leave TheHive out entirely when I went through the installation. I did a Standalone server.

**Comment by defensivedepth** [comment, marked as answer]
@danmessner1 Thanks for letting us know. What would be really helpful is if you posted the setup log (`/root/sosetup.log`) for any failed installs that you ran into. This would make it alot easier to see what is going on.
